src/rospy_message_converter/message_converter.py

```diff
@@ -146,7 +146,7 @@ def _convert_to_ros_time(field_type, field_value):
     return time
 
 def _convert_to_ros_primitive(field_type, field_value):
-    if field_type == "string":
+    if field_type == "string" and not python3:
```

Comment:
Perhaps add a small comment here, something along the lines of:

```suggestion
    # std_msgs/msg/_String.py already calls encode() on python3, so don't do it here
    if field_type == "string" and not python3:
```